airflow/www/templates/airflow/dags.html

```diff
@@ -28,6 +28,17 @@
 {% block content %}
   <h2>DAGs</h2>
 
+{% if not is_user_logged_in %}
+<div class="alert alert-warning" role="alert">
+  Note: Some DAGs may not be visible until you <a href="/login/">log in</a>.
```

Comment:
When is this going to get shown? Anon users can't get here can they?

Reply:
They will be able to, actually.

FAB's BaseSecurityManager's implementation of `has_access` allows unauthenticated users to access any views that are accessible by the public role (see [1], which calls [2])

[1] https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/security/manager.py#L894
[2] https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/security/manager.py#L868

Reply:
But we don't have any views publicly accessible do we?